src/main/java/core/basesyntax/FigureArea.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface FigureArea {
+    public abstract double areaCalculator();
+}
```

Comment:
Interface names should not be verbs or contain verbs. Consider renaming the interface to something that represents a characteristic or capability, such as 'AreaCalculable' to follow Java naming conventions.